### PR TITLE
Add F# example for AssemblyBuilder class.

### LINKED ISSUE #7719: Missing F# code sample for AssemblyBuilder class.
The [page on the AssemblyBuilder class](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.assemblybuilder?view=net-6.0) is missing an F# code sample, despite having the option for the language.

### PR DESCRIPTION
**Pull request by brenoafb**
## Summary

Adds F# code sample for the [AssemblyBuilder class](https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.assemblybuilder?view=net-6.0).

The code is pretty much 1:1 with the C# counterpart, aside from some variable names;
the result is identical.

Fixes #7719

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=7720) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=7720) before we can accept your contribution.<br/><br/>:x: brenoafb [sign now](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=7720)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/dotnet-api-docs?pullRequest=7720) it.</sub>

File: xml/System.Reflection.Emit/AssemblyBuilder.xml [hunk 1]
```diff
@@ -64,52 +64,53 @@
...
+
+## Examples
 The following code example shows how to define and use a dynamic assembly. The example assembly contains one type, `MyDynamicType`, that has a private field, a property that gets and sets the private field, constructors that initialize the private field, and a method that multiplies a user-supplied number by the private field value and returns the result.
 
  :::code language="cpp" source="~/snippets/cpp/VS_Snippets_CLR/AssemblyBuilderClass/cpp/24895.cpp" id="Snippet1":::
  :::code language="csharp" source="~/snippets/csharp/System.Reflection.Emit/AssemblyBuilder/Overview/24895.cs" id="Snippet1":::
- :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR/AssemblyBuilderClass/vb/24895.vb" id="Snippet1":::  
-  
+ :::code language="fsharp" source="~/snippets/fsharp/System.Reflection.Emit/AssemblyBuilder/Overview/24895.fs" id="Snippet1":::
```
Comment (MSDN-WhiteKnight) [inline xml/System.Reflection.Emit/AssemblyBuilder.xml]: You referenced a Snippet1, but didn't define an actual named snippet in source (which is done by syntax like `//<Snippet1> ... //</Snippet1>`). Consider dropping `id="Snippet1"` part if you want to just reference a whole source file.

Comment (brenoafb) [reply xml/System.Reflection.Emit/AssemblyBuilder.xml:111]: Thanks a lot! Fixed it.

**Review by dsyme — APPROVED** [review]

**Review by BillWagner — APPROVED** [review]
Thanks for adding all these samples @brenoafb 

I'll :shipit: now.

You should see the changes in the live site in the next day or two, on our regular publishing cycle.

**Pull request merged**